samples/estimation/ShorRE.qs

```diff
@@ -0,0 +1,203 @@
+/// # Sample
```

Comment:
A quantum-only Shor sample has been in the plan for more advanced algorithmic sample (#767). Since this sample is exactly that, would it be make sense to move it to the algorithms folder as `ShorEstimateFrequency` or something similar? Or is the objective to have a specific set of samples dedicated to RE only and those are expected to be in the `estimation` folder to be able to refer people to it?

Reply:
For now the intent is to have this only be for RE. As configured, it's too big to simulate reasonably, and the entry point doesn't return any values. I think keeping this as estimation specific might make more sense until we have a better plan for what we want out of the quantum only version.